src/components/auth/Register.js

```diff
@@ -42,60 +41,80 @@ const Styles = styled.div`
         width: 450px;
     }
 
-    .form-select{
-        padding: 10
+    .form-select {
+        padding: 8px;
     }
 
-    .logo{
-        height: 80px
+    .logo {
+        height: 80px;
     }
 
     .logo:hover {
         cursor: pointer;
     }
 
-    .pinwheel-wrapper{  
+    .pinwheel-wrapper {
         display: flex;
         justify-content: center;
     }
-    .pinwheel{
-        height: 200px
+    .pinwheel {
+        height: 200px;
     }
 
-    .form-title{
+    .other-specification {
+        margin-left: 50px;
+    }
+
+    .form-title {
         text-align: center;
         font-weight: 900;
         font-size: 2rem;
-        padding: 15px
+        padding: 15px;
     }
 
-    .form-success-title{
+    .form-success-title {
         text-align: center;
         font-weight: 900;
         font-size: 2rem;
-        padding-top: 80px; 
-        padding-bottom: 10px
+        padding-top: 80px;
+        padding-bottom: 10px;
     }
 
-    .stepper{
-        height: 50px;
-        maxWidth: 400;
-        flexGrow: 1;
-        color: blue
+    .form-description {
+        text-align: center;
+        font-weight: 500;
+        font-size: 1.3rem;
+        padding-bottom: 20px;
+    }
+
+    .form-input-description {
+        font-size: 0.95rem;
+        text-align: center;
+        padding-bottom: 10px;
     }
 
+    .stepper {
+        height: 50px;
+        maxwidth: 400;
+        flexgrow: 1;
+        color: blue;
+    }
 
     .next-button {
-        background: rgb(1,131,225);
-        background: linear-gradient(90deg, rgba(1,131,225,1) 0%, rgba(90,184,253,1) 100%);
+        background: rgb(1, 131, 225);
+        background: linear-gradient(
+            90deg,
+            rgba(1, 131, 225, 1) 0%,
+            rgba(90, 184, 253, 1) 100%
+        );
         border-radius: 12px;
         border: none;
         height: 50px;
         font-size: 1.3rem;
         font-weight: 600;
     }
```

Comment:
This button looks beautiful! Can you make it a separate component, so we can re-use on other parts or the website?

Reply:
Ofc